### DISCUSSION TITLE
Quizz eval

### DISCUSSION THREAD

**Discussion by JarodStreck** [web-classroom/web-classroom.github.io#10] [Q&A : Théorie / Cours]
Bonjour,

Nous avons fait 2 Quizz sur eval.iict-heig-vd.in . Et malgrés mon e-mail du 18 Octobre, nous n'avons toujours pas à disposition les questions des Quizz afin de réviser.

**Comment by OlivierLmr** [comment]
Bonjour, votre mail s'était effectivement perdu dans les autres réponses que j'avais reçu à mon mail du 18. N'hésitez pas à me relancer plus tôt dans le futur si cela devait se reproduire.

Les questions et leurs solutions sont par contre déjà bien disponibles sur les slides, là où les quiz ont eu lieu. Si vous ne les voyez pas c'est qu'il y a un problème quelque part, et je vous les transmettrai par un autre moyen.

J'en profite d'ailleurs pour rappeler que ces quiz sont des exercices dont le but est de consolider les sujets vus en cours en vous les faisant travailler de manière poussée, et non des tests blancs. Le niveau de certaines de ces questions n'est donc pas représentatif de celui du test de demain.
